examples/bq_file_load_benchmark/benchmark_tools/staging_table_generator.py

```diff
@@ -0,0 +1,221 @@
+            command = command_str.format(
+                data_gen_path,
+                self.json_schema_path,
+                schema_name,
+                self.num_rows,
+                self.project,
+                self.staging_dataset_id,
+                dataflow_staging_location,
+                dataflow_temp_location,
+            )
+            os.system(command)
+
+    def create_resized_tables(self):
+        """Creates resized staging tables using the targetDataSizes parameters.
+
+        Utilizes the bq_table_resizer module from the Dataflow Data
+        Generator tool to create resized staging tables.
+
+        """
+        staging_dataset_ref = self.bq_client.dataset(self.staging_dataset_id)
+        staging_dataset = bigquery.Dataset(staging_dataset_ref)
+        sizes = self.file_params['targetDataSizes']
+        sizes.sort()
+        abs_path = os.path.abspath(os.path.dirname(__file__))
+        bq_resizer_path = os.path.join(
+            abs_path,
+            '../../dataflow-data-generator/bigquery-scripts'
+        )
+        # Gather staging tables that were created in
+        # self.create_staging_tables()
+        tables = list(self.bq_client.list_tables(staging_dataset))
+        for table in tables:
+            for i in range(0, len(sizes)):
+                # If the size of the current iteration is the smallest size
+                # in the sizes list, use the corresponding staging table
+                # created in create_resized_table() as the source base table.
+                # Otherwise, if the file size of the current iteration is
+                # greater than the last iteration, use the resized table from
+                # the previous iteration as the source base table when running
+                # so that each table can take advantage of the size of the last.
+                size = sizes[i]
+                if size > sizes[i - 1]:
+                    base = sizes[i - 1]
+                    source_table = get_resized_table_name(
+                        table.table_id,
+                        base
+                    )
+                    source_dataset = self.resized_dataset_id
+                else:
+                    source_table = table.table_id
+                    source_dataset = self.staging_dataset_id
+
+                destination_table = get_resized_table_name(
+                    table.table_id,
+                    size
+                )
+                target_gb = size
+                command_str = ('python {0:s}/bq_table_resizer.py '
+                               '--project {1:s} '
+                               '--source_dataset {2:s} '
+                               '--source_table {3:s} '
+                               '--destination_dataset {4:s} '
+                               '--destination_table {5:s} '
+                               '--target_gb {6:f} ')
+                command = command_str.format(
```

Comment:
You don't use this unformatted `command_str` ever so just add this format to the last statement.